apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -184,7 +184,7 @@ func verifyReplicaCount(incomingSo *ScaledObject, action string, _ bool) error {
 		scaledobjectlog.WithValues("name", incomingSo.Name).Error(err, "validation error")
 		metricscollector.RecordScaledObjectValidatingErrors(incomingSo.Namespace, action, "incorrect-replicas")
 	}
-	return nil
+	return err
```

Comment:
hmm, @JorTurFer, @zroubalik, any thoughts why the webhook used to allow wrong min/max? I just tried it with 2.16.0 and I can easily create `SO` with `min=25` and `max=10`.
```
2024-12-02T14:09:36Z    ERROR   scaledobject-validation-webhook validation error        {"name": "http-server", "error": "MinReplicaCount=25 must be less than MaxReplicaCount=10"}
```

Reply:
Probably it was a mistake, the admission webhook should prevent that clear misconfiguration